var/spack/repos/builtin/packages/py-ytopt/package.py

```diff
@@ -28,7 +29,7 @@ class PyYtopt(PythonPackage):
     depends_on('py-dh-scikit-optimize', type=('build', 'run'))
     depends_on('py-configspace', type=('build', 'run'))
     depends_on('py-numpy', type=('build', 'run'))
-    depends_on('py-ytopt-autotune@1.1:', type=('build', 'run'))
+    depends_on('py-ytopt-autotune@1.0.0', type=('build', 'run'))
```

Comment:
You're right this would allow bugfixes in v1.0, @alalazo,  would this work?
```suggestion
    depends_on('py-ytopt-autotune@1.0.0:1.0.999', type=('build', 'run'))
```

Reply:
Sure, this would work fine, thanks!